app/controllers/Contributions.scala

```diff
@@ -89,12 +90,17 @@ class Contributions(paymentServices: PaymentServices, addToken: CSRFAddToken) ex
     val charge = request.session.get("charge_id")
     val title = "Thank you!"
 
+    val iosRedirectUrl = request.session.get("amount")
```

Comment:
My personal preference would be to use a `for` comprehension. Easy either way though.